src/components/inputs/date-input/date-input.js

```diff
@@ -1,357 +1,271 @@
-/*
-  TECHDEBT:
-  - Support timeZone in all cases, but only when timeZone is passed.
-*/
-
-import 'flatpickr/dist/themes/airbnb.css';
 import React from 'react';
 import PropTypes from 'prop-types';
-import requiredIf from 'react-required-if';
+import Downshift from 'downshift';
```

Comment:
Note that we're not importing `moment` into this component at all. All usage of `moment` is moved to the utils. This allows to use the `date-input` (and the other ones) with any date-library (like [`date-fns`](https://github.com/date-fns/date-fns)).